src/features/zaaksysteem/ZakenOverzicht.vue

```diff
@@ -25,9 +29,15 @@
 
 <script lang="ts" setup>
 import { formatDateOnly } from "@/helpers/date";
-import type { Zaak } from "@/stores/contactmoment";
 import type { PropType } from "vue";
 import ZaakContactmomentKoppelaar from "./zaakContactmomentKoppelaar.vue";
+import type { Zaak } from "./types";
+
+const emit = defineEmits(["ZaakSelected"]);
```

Comment:
I see that (almost) all emits within the project are camelCased, this one starts with an upper. Might be good to also make camel cased for consistency.

Reply:
done